datasets/BEIS-Energy-Trends-Table-1-2-Extract/info.json

```diff
@@ -5,46 +5,43 @@
     "publisher": "Department for Business, Energy and Industrial Strategy",
     "description": "",
     "landingPage": "https://www.gov.uk/government/statistics/total-energy-section-1-energy-trends",
+    "dataURL": "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1173868/ET_1.2_JUL_23.xlsx",
     "datasetNotes": [
-        "No landing page for this dataset as it is a bespoke dataset supplied by BEIS (Warren Evans) directly to IDS-D as part of the Climate Change Platform project. It is an extract of BEIS published data of the same name, landing page: https://www.gov.uk/government/statistics/total-energy-section-1-energy-trends"
+        "On landing page, navigate to the spreadsheet 'Inland energy consumption: primary fuel input basis (ET 1.2 - monthly)'"
     ],
-    "published": "24-02-2022",
+    "published": "2023-08-27",
     "families": [
         "Climate-Change"
     ],
     "extract": {
-        "source": "CSV"
+        "source": "XLSX"
     },
     "transform": {
-        "airtable": "recZeJg9wyH5NBPXN",
-        "main_issue": 47,
+        "airtable": "rect0gFc8uVpxFsa9",
+        "main_issue": 523,
         "columns": {
-            "Period": {
+            "Year": {
```

Comment:
We need the sub-property to be to get the filter

Reply:
Thanks and sorted